src/main/java/org/example/Main.java

```diff
@@ -1,17 +1,38 @@
 package org.example;
```

Comment:
Could you change the naming of the package to be more descriptive than this one?

Reply:
done :)